### PR TITLE
Add CH release notes

### PR DESCRIPTION
**Pull request by aiannucci**

### CODE REVIEW

File: modules/ROOT/pages/cloudhub/cloudhub-runtimes-release-notes.adoc [hunk 1]
```diff
@@ -4,16 +4,33 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: release notes, cloudhub, cloud hub
 
-These are the release notes for patches which are made to the Mule runtimes on CloudHub. In addition to these release notes, see the xref:cloudhub/cloudhub-release-notes.adoc[CloudHub release notes] and complete xref:runtime-manager::cloudhub.adoc[CloudHub] documentation.
+These are the release notes for patches made to the Mule runtimes on CloudHub. In addition to these release notes, see the xref:cloudhub/cloudhub-release-notes.adoc[CloudHub release notes] and complete xref:runtime-manager::cloudhub.adoc[CloudHub] documentation.
 
-== May 22, 2019
+== May 24, 2019
+
+4.1.4 and 4.2.0 Runtime Update:
```
Comment (eugene-ivanov-mulesoft) [inline modules/ROOT/pages/cloudhub/cloudhub-runtimes-release-notes.adoc]: 4.1.5 and 4.2.0

File: modules/ROOT/pages/cloudhub/cloudhub-runtimes-release-notes.adoc [hunk 2]
```diff
@@ -4,16 +4,33 @@ include::_attributes.adoc[]
...
 
-These are the release notes for patches which are made to the Mule runtimes on CloudHub. In addition to these release notes, see the xref:cloudhub/cloudhub-release-notes.adoc[CloudHub release notes] and complete xref:runtime-manager::cloudhub.adoc[CloudHub] documentation.
+These are the release notes for patches made to the Mule runtimes on CloudHub. In addition to these release notes, see the xref:cloudhub/cloudhub-release-notes.adoc[CloudHub release notes] and complete xref:runtime-manager::cloudhub.adoc[CloudHub] documentation.
 
-== May 22, 2019
+== May 24, 2019
+
+4.1.5 and 4.2.0 Runtime Update:
+
+* Visuzlizer properties not picked up from the `mule-app.properties` file [SE-10374
```
Comment (eugene-ivanov-mulesoft) [inline modules/ROOT/pages/cloudhub/cloudhub-runtimes-release-notes.adoc]: missing ]

File: modules/ROOT/pages/cloudhub/cloudhub-runtimes-release-notes.adoc [hunk 3]
```diff
@@ -4,16 +4,33 @@ include::_attributes.adoc[]
...
+
+3.9.2 Runtime Update:
+
+* Address incomplete fix in OAuth service [SE-10423]
+* Visuzlizer properties not picked up from the `mule-app.properties` file [SE-10374
```
Comment (eugene-ivanov-mulesoft) [inline modules/ROOT/pages/cloudhub/cloudhub-runtimes-release-notes.adoc]: missing ]

File: modules/ROOT/pages/cloudhub/cloudhub-runtimes-release-notes.adoc (hunk 2 above)
Comment (valkyrie69) [reply modules/ROOT/pages/cloudhub/cloudhub-runtimes-release-notes.adoc]: Oh my goodness! :-D

**Pull request merged**